projects/aca-shared/rules/src/app.rules.spec.ts

```diff
@@ -1207,58 +1032,35 @@ describe('app.evaluators', () => {
     });
   });
 
-  describe('isTrashcanItemSelected', () => {
-    it('should return false when user is not in trashcan', () => {
-      context.navigation.url = '/personal-files/test';
-      expect(app.isTrashcanItemSelected(context)).toBeFalse();
-    });
-
-    it('should return false when nothing is selected', () => {
-      context.navigation.url = '/trashcan/test';
-      context.selection.isEmpty = true;
-      expect(app.isTrashcanItemSelected(context)).toBeFalse();
-    });
-
-    it('should return true when user is in trashcan and node is selected', () => {
-      context.navigation.url = '/trashcan/test';
-      context.selection.isEmpty = false;
-      expect(app.isTrashcanItemSelected(context)).toBeTrue();
-    });
-  });
-
-  describe('canViewFile', () => {
-    it('should return false when user is in trashcan', () => {
-      context.navigation.url = '/trashcan/test';
-      expect(app.canViewFile(context)).toBeFalse();
-    });
-
-    it('should return false when nothing is selected', () => {
-      context.navigation.url = '/personal-files/test';
-      context.selection.file = null;
-      expect(app.canViewFile(context)).toBeFalse();
+  describe('canToggleFileLock', () => {
+    beforeEach(() => {
+      context.profile = {} as any;
     });
 
-    it('should return true when user is not in trashcan and file is selected', () => {
-      context.navigation.url = '/personal-files/test';
-      context.selection.file = {} as any;
-      expect(app.canViewFile(context)).toBeTrue();
+    it('should return false when permission requirements are not met, regardless of file lock', () => {
+      context.selection.file = { entry: { properties: { 'cm:lockType': 'WRITE_LOCK', 'cm:lockOwner': { id: 'test' } } } } as any;
```

Comment:
please use "as NodeEntry" instead of "as any"

Reply:
Replaced any with NodeEntry. However, looking at the file, there are a lot more instances of any in this file (137 to be exact). Most of them can be replaced with NodeEntry itself, but I feel that making that change in this PR would make it even more difficult to review. Should we raise a separate task for it?

Reply:
sure, you can raise separate task for this, thank you